packages/github/tsconfig.json

```diff
@@ -2,7 +2,6 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "baseUrl": "./",
-    "esModuleInterop": true,
```

Comment:
moved up to the root tsconfig.json

i left a comment on the root tsconfig.json which explains this more